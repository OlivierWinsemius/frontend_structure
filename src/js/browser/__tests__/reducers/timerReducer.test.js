import timer from '../../browserReducers/timerReducer'

describe('Timer Reducer', () => {
    it('has a default state', () => {
        expect(timer()).toMatchSnapshot();
    })
});
