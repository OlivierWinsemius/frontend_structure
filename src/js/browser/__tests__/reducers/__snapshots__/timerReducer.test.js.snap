// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer Reducer has a default state 1`] = `
Object {
  "startedAt": null,
  "stoppedAt": null,
}
`;
